client/ui/components/AddonGroupForm.jsx

```diff
@@ -138,58 +129,59 @@ class AddonSelectField extends Component {
 
 class AddonGroupForm extends React.Component {
   static propTypes = {
-    activeAddonGroup: pt.object.isRequired,
-    addonsList: pt.object.isRequired,
-    createAddonGroup: pt.object.isRequired,
-    fetchAddonGroup: pt.func.isRequired,
-    fetchAddons: pt.func.isRequired,
+    addonGroup: pt.object,
+    addons: pt.array,
+    fetchRequest: pt.object,
     handleSubmit: pt.func.isRequired,
-    pk: pt.any,
-    resetAll: pt.func.isRequired,
-    save: pt.func.isRequired,
-    saveAndContinue: pt.func.isRequired,
+    initialize: pt.func.isRequired,
```

Comment:
[Redux-Form exports propTypes for these](https://github.com/erikras/redux-form/blob/master/src/propTypes.js) in case you want to use that instead.